src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx

```diff
@@ -85,8 +86,14 @@ const DocumentUploader = () => {
         <FileList files={state.files} onClick={onDeleteClick} />
       )}
       {state.successMessage ? (
-        <va-alert status="success" visible>
-          <h3 slot="headline">Your files have been uploaded</h3>
+        <va-alert
+          background-only
+          close-btn-aria-label="Close notification"
+          show-icon
+          status="success"
+          visible
+        >
+          <div>Your files have been uploaded</div>
```

Comment:
Should probably be a `<p>`

Reply:
@timwright12 I agree, I will update this. Interestingly I copied this code from the storybook on alerts so we might want to update the example code and when you change it to a `p` tag the padding is off from the styles inherited from paragraph tags on our site https://design.va.gov/storybook/?path=/docs/components-va-alert--background-only-with-icon 

<img width="674" alt="Screen Shot 2022-03-10 at 10 13 37 AM" src="https://user-images.githubusercontent.com/1899695/157728347-2e2c6cee-6509-4177-bd9f-61fac03059db.png">